### PR TITLE
Use Video Android 7.0.0

### PR DESCRIPTION
**Pull request by mmartikainen**
#### New Features

- The SDK has been upgraded to use Chromium WebRTC 88.
- The SDK uses Unified Plan SDP semantics instead of Plan-B. This brings several important benefits:
  - Improved interoperability with Firefox, Safari and Chrome in Peer-to-Peer and Go Rooms.
  - Track level operations like publishing and unpublishing are more reliable.
- Improved performance of `Room.getStats()` by adopting the W3C standardized RTC stats API.
- Unused codecs for a track are removed from the local SDP offer in order to reduce the SDP size, once codecs for the
  track have been negotiated.

#### API Changes

- The minimum Android SDK version has been increased from 16 to 21. Please visit our [migration guide](https://www.twilio.com/docs/video/migrating-6x-7x) for more details on how to update your application to 7.0!
- The ordering of `RemoteParticipantObserver.onAudioTrackSubscribed()`
  and `RemoteParticipantObserver.onVideoTrackSubscribed()` is not strictly guaranteed. In this
  release the ordering of these callbacks may be different when compared to previous releases.
- The value of `LocalVideoTrackStats.frames_encoded` now reflects the total number of frames encoded for a simulcast track.

#### Bug Fixes

- Fixed a bug causing the Connect Option `enableIceGatheringOnAnyAddressPorts` not to have any effect.

#### Known Issues

- As of 6.0.0 of the SDK, hardware video encoding doesn't publish all of the three simulcast layers when VP8 simulcast is enabled on Android.
  This affects Video Content Preferences from working properly for subscribing video participants since the feature requires all three simulcast layers to switch between.
  Our team is working on a fix for this, but the feature does work when VP8 simulcast is used to publish video from participants using the Javascript or iOS SDKs.
- Video Content Preferences might prefer larger video than needed when device orientations are mismatched. For example, if a participant in landscape mode publishes video, then the subscribing participant must also be in landscape mode in order for the correctly sized simulcast layers to be selected. The same is true for the portrait orientation.
- When the publisher is publishing video at 720p with VP8 simulcast enabled and the subscriber varies their hints between 180p, 360p, and 720p, sometimes the subscriber receives larger video than expected.

**Contributing to Twilio**

> All third-party contributors acknowledge that any contributions they provide will be made under the same open-source license that the open-source project is provided under.

- [x] I acknowledge that all my contributions will be made under the project's license.

### CODE REVIEW

**Comment by mmartikainen** [comment]
**Note:** We should wait for the migration guide link to go live before we merge.

**Review by aaalaniz — APPROVED** [review]

**Pull request merged**
